### ISSUE TITLE
hover not working on chrome

### ISSUE THREAD

**Issue by reczuprik** [kevin-powell/space-tourism#1]
Hello Kevin! thanks for the course.
issue i found that on Chrome; Version 95.0.4638.54 (Official Build) (64-bit) 
.dot-indicators>*:hover or .underline-indicators>*:hover not working (chrome does not detect it), and focus only works on tab-list and not on primary-navigation

I tried based on the videos and also downloaded the version from here

**Comment by kevin-powell** [comment]
I've updated all the `design-system.html` files throughout the repo. It was a missing `<!DOCTYPE html>` in that file. 

Turns out, when the doctype is missing, browsers enter _quirks_ mode, and that behaviour is part of the spec of quirks mode. 

I didn't catch it when I created the file originally because the Scrimba in-built preview didn't have the same issue for some reason. Might be from it using an iframe, though I'm not 100% sure.
